docker/README.md

```diff
@@ -1,86 +1,74 @@
-# Docker images for Autoware
+# Open AD Kit: Containerized Workloads for the Autoware
```

Comment:
Just a reminder to align with https://github.com/autowarefoundation/autoware-documentation/pull/516#discussion_r1511126441

Reply:
done (can be resolved)